app/services/xccdf_report_parser.rb

```
@@ -13,7 +13,7 @@ class XccdfReportParser
   attr_reader :report_path, :test_result_file
 
   def initialize(report_contents, message)
-    raise ::EmptyMetadataError if message['metadata'].blank?
+    raise ::MessageFormatError unless valid_message_format?(message)
```

We also need to raise an error if the host is not found in the inventory.